unified-runtime/source/common/CMakeLists.txt

```diff
@@ -36,41 +36,41 @@ target_include_directories(ur_common PUBLIC
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
 )
 
-message(STATUS "Download Unified Memory Framework from github.com")
-if (NOT DEFINED UMF_REPO)
-    set(UMF_REPO "https://github.com/oneapi-src/unified-memory-framework.git")
-endif()
-
-if (NOT DEFINED UMF_TAG)
-    # commit 1de269c00e46b7cbdbafa2247812c8c4bb4ed4a5
-    # Author: Łukasz Stolarczuk <lukasz.stolarczuk@intel.com>
-    # Date:   Mon Jul 21 15:42:59 2025 +0200
-    # 1.0.0 release
-    set(UMF_TAG v1.0.0)
-endif()
-
-message(STATUS "Will fetch Unified Memory Framework from ${UMF_REPO}")
-
-include(FetchContent)
-FetchContent_Declare(unified-memory-framework
-    GIT_REPOSITORY    ${UMF_REPO}
-    GIT_TAG           ${UMF_TAG}
-)
 
 if (UR_STATIC_ADAPTER_L0)
     if (UMF_BUILD_SHARED_LIBRARY)
         message(STATUS "Static adapter is not compatible with shared UMF, switching to fully statically linked UMF")
         set(UMF_BUILD_SHARED_LIBRARY OFF)
     endif()
 endif()
+  
+set(UR_USE_EXTERNAL_UMF ON CACHE BOOL "Use a pre-built UMF if available")
 
-set(UR_USE_EXTERNAL_UMF OFF CACHE BOOL "Use a pre-built UMF")
-
-if (UR_USE_EXTERNAL_UMF)
-  find_package(umf REQUIRED)
+if(UR_USE_EXTERNAL_UMF)
+  find_package(umf QUIET)
```

Comment:
we could specify looking for 1.0.0 version (or newer)

I'm afraid without this we can have a lot of random issues with version incompat., especially given our recent change from v0.12.x to v1.0.x

Reply:
Sure, I'll add the version check, thanks!